skimage/feature/corner.py

```diff
@@ -906,7 +905,7 @@ def corner_subpix(image, corners, window_size=11, alpha=0.99):
         maxx = x0 + wext + 2
         window = image[miny:maxy, minx:maxx]
 
-        winx, winy = _compute_derivatives(window, mode='constant', cval=0)
+        winy, winx = _compute_derivatives(window, mode='constant', cval=0)
 
         # compute gradient suares and remove border
```

Comment:
suares = squares?

Reply:
To keep the diff small, I tried to avoid changes that weren't germane to #4831. Happy to make drive-by fixes if that's desired.